brew/vars.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -eu
+
+# TODO(mkramlich): commit hash/pin (for brew formula)
+# TODO(mkramlich): elim redundancy with or otherwise maybe take advantage of version/version.sh
+
+cat <<EOF
```

Comment:
This is a very weird way to do a whole bunch of export statements in a shell script. If you remove this heredoc and the other headers you will be able to `source` the file instead of having to evaluate it.

Reply:
yeah a variety of ways of doing it. the eval() technique has the advantage of letting you use any language, and, in a certain way you get a tad better isolation. totally see attraction of source! It was like a 5 second decision. I did a quick scan of the existing scripts in the repo and none used 'source'. The build_dist.sh uses this exact eval() technique so I assumed it was approved. and makes brew more consistent with it. I assumed in future revisions there would be more automation and tighter integration with the rest of the release system, so it seemed like a good direction.